### PR TITLE
Add *description* property to nuclei-templates

### PR DESCRIPTION
**Pull request by toufik-airane**
# Description

Recently, the update https://github.com/projectdiscovery/nuclei/pull/140 introduces a property in the nuclei template engine.
The property named **description** gives space to describe the result of a template.

However, most of the actual templates don't implement it.

# About the contribution

## Type of change

- [X] Non-breaking changes

### CODE REVIEW

**Review by toufik-airane — COMMENTED** [review]
@bauthard @ioribrn can we move with that first bulk?

Thanks.

File: vulnerabilities/cached-aem-pages.yaml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 id: cached-pages-aem
 
 info:
-  name: Invalidate / Flush Cached Pages From AEM
+  name: Invalidate / Flush Cached Pages on AEM
   author: hetroublemakr
   severity: low
-  # reference: @AEMSecurity
+  description: todo
+  reference: https://twitter.com/AEMSecurity
```
Comment (bauthard) [inline vulnerabilities/cached-aem-pages.yaml:8]: @toufik-airane `reference` field is something still needs to be added, can we comment it out for now?

File: vulnerabilities/ibm-infoprint-directory-traversal.yaml [hunk 2]
```diff
@@ -3,9 +3,9 @@ id: IBM InfoPrint Directory Traversal
 info:
   name: IBM InfoPrint 4247-Z03 Impact Matrix Printer - Directory Traversal
   author: Harsh Bothra
-  severity: Medium
-
-# source:- https://www.exploit-db.com/exploits/47835
+  severity: medium
+  description: Directory traversal vulnerability on IBM InfoPrint 4247-Z03 Impact Matrix Printer.
+  reference: https://www.exploit-db.com/exploits/47835
```
Comment (bauthard) [inline vulnerabilities/ibm-infoprint-directory-traversal.yaml:8]: same as above, once we add `reference`, we can remove the #

File: vulnerabilities/microstrategy-ssrf.yaml [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-  name: MicroStrategy tinyurl SSRF
+  name: MicroStrategy tinyurl - BSSRF
   author: organiccrap
   severity: high
-  # https://medium.com/@win3zz/how-i-made-31500-by-submitting-a-bug-to-facebook-d31bb046e204
-  # blind ssrf
-  # /servlet/taskProc?taskId=wikiScrapper&taskEnv=xml&taskContentType=xml&searchString=https://SSRF&shouldSuggest=false&publicDataSuggestionURL=&publicDataSearchURL=&publicDataPageURL=
-  # /servlet/taskProc?taskId=validateUsherConfigTask&taskEnv=xml&taskContentType=xml&serverURL=https://SSRF
+  description: Blind server-side request forgery vulnerability on MicroStrategy URL shortener.
+  reference: https://medium.com/@win3zz/how-i-made-31500-by-submitting-a-bug-to-facebook-d31bb046e204
```
Comment (bauthard) [inline vulnerabilities/microstrategy-ssrf.yaml:7]: This as well.

File: vulnerabilities/symfony-debugmode.yaml [hunk 4]
```diff
@@ -3,7 +3,8 @@ info:
   name: Symfony Debug Mode
   author: organiccrap
   severity: high
-  # https://github.com/synacktiv/eos
+  description: todo
+  reference: https://github.com/synacktiv/eos
```
Comment (bauthard) [inline vulnerabilities/symfony-debugmode.yaml:7]: `# reference`

**Review by bauthard — CHANGES_REQUESTED** [review]

**Comment by toufik-airane** [comment]
> @bauthard @ioribrn can we move with that first bulk?
> 
> Thanks.

Done.

**Comment by bauthard** [comment]
Thank a lot @toufik-airane @ioribrn for adding this useful information.

**Comment by ioribrn** [comment]
Thanks @bauthard

**Pull request merged**